docs/conf.py

```diff
@@ -36,3 +36,14 @@ def setup(app):
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
+
+# Custom Alabaster theme options
+html_theme_options = {
+    'description': 'A simple JupyterHub distribution for 1-100 users',
+    'github_user': 'jupyterhub',
+    'github_repo': 'the-littlest-jupyterhub',
+    'github_button': 'true',
+    'extra_nav_links': {
+        'Documentation confusing? File an issue!': 'https://github.com/jupyterhub/the-littlest-jupyterhub/issues'
```

Comment:
Needs something like

```
html_sidebars = {
    '**': [
        'about.html',
        'navigation.html',
        'relations.html',
        'searchbox.html',
        'donate.html',
    ]
}
```

To be set, but fuck up the layout and put a link to all the pages.

Reply:
<img width="1010" alt="screen shot 2018-07-29 at 11 51 23" src="https://user-images.githubusercontent.com/335567/43369573-bf2cfe86-9325-11e8-9239-b705df1611f6.png">